Foundation/NSTimeZone.swift

```diff
@@ -29,17 +29,13 @@ open class NSTimeZone : NSObject, NSCopying, NSSecureCoding, NSCoding {
     }
 
     public init?(name tzName: String, data aData: Data?) {
-#if os(Android) || os(Windows)
+#if os(Windows)
```

Comment:
:-(

Reply:
I tried to not modify the Windows code. I was under the impression that you added support for Windows. Is this a left over?

Reply:
No, this implies that we have more work to do on Windows and android is currently doing better than Windows in the Time Zone department.